### ISSUE TITLE
ATLauncher pack icons are crushed when running

### ISSUE THREAD

**Issue by r58Playz** [MultiMC/MultiMC5#3820]
### Operating System

macOS

### Description of bug

Modpack icons are crushed when shown in the dock on MacOS. Maybe it occurs on Windows and Linux too.

Curseforge instance icon when running:
<img width="42" alt="Screen Shot 2021-05-28 at 5 23 18 PM" src="https://user-images.githubusercontent.com/45221816/120052312-972c3f80-bfd9-11eb-8446-e4696208d2a4.png">
ATLauncher instance icon when running:
<img width="42" alt="Screen Shot 2021-05-28 at 5 22 50 PM" src="https://user-images.githubusercontent.com/45221816/120052314-97c4d600-bfd9-11eb-8d1a-be0e741336b5.png">

### Steps to reproduce

Download an ATLauncher modpack and run it.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by ImperatorStorm** [comment]
Can't reproduce on linux using KDE Plasma DE. For whatever reason, custom pack icon isn't used at all:
![image of taskbar icon](https://i.imgur.com/S6R7mN9.png)
This is despite using the same pack as @r58Playz

**Comment by jamierocks** [comment]
Yeah this is an unfortunate consequence of ATLauncher using rectangular pack icons.

I guess the logos could be set onto a square background, to prevent them being stretched in cases like this - though that would look equally bad imo, if not worse.
